neptunecontrib/create_experiment_comparison_comment.py

```diff
@@ -110,9 +110,11 @@ def create_comment_markdown(df, project_name):
         if 'property_' in k:
             data['properties'][k.replace('property_', '')] = [v[0], v[1]]
 
+    user, project = project_name.split('/')
+
     # link to experiment comparison
-    link = "https://ui.neptune.ai/o/shared/org/github-actions/compare?shortId=%5B%22{}%22%2C%22{}%22%5D".format(
-        data['id'][0], data['id'][1])
+    link = "https://ui.neptune.ai/o/{0}/org/{1}/compare?shortId=%5B%22{2}%22%2C%22{3}%22%5D".format(
```

Comment:
This should be `https://ui.neptune.ai/{0}/{1}/compare?shortId=%5B%22{2}%22%2C%22{3}%22%5D`

Reply:
Done